module/os/windows/zfs/zfs_vnops_os.c

```diff
@@ -3081,6 +3081,9 @@ zfs_rename(znode_t *sdzp, char *snm, znode_t *tdzp, char *tnm,
 	int		zflg = 0;
 	boolean_t	waited = B_FALSE;
 
+	if (rflags != 0 || wo_vap != NULL)
+		return (SET_ERROR(EINVAL));
+
```

Comment:
Ah ok, we are saying if the new features are used, just bail - presumably "for now". Later we can check if there are Windows like calls (and I think there was something in the open-extra-options), and add to it.

Reply:
Yeah that was exactly the idea